app/src/components/utils.js

```diff
@@ -66,6 +69,25 @@ const utils = {
       return error.response.data;
     }
   },
+  /**
+  * @function getClientsFromEnv
+  * Utility function to call the KC service to get clients for each realm which requires newing it for each realm
+  * @param {string} kcEnv The KC env
+  * @param {string} acronyms The acronyms to get clients for
+  * @returns {object[]} An array of service clients
+  */
+  getClientsFromEnv: async (kcEnv, acronyms) => {
+    const realmKey = `serviceClient.keycloak.${kcEnv}`;
+    const {
+      endpoint: realmBaseUrl,
+      username: clientId,
+      password: clientSecret,
+      realm: realmId
+    } = config.get(realmKey);
+    const realmSvc = new RealmAdminService({ realmBaseUrl, clientId, clientSecret, realmId });
+    const kcScMgr = new KeyCloakServiceClientManager(realmSvc);
+    return kcScMgr.fetchClients(acronyms);
+  }
```

Comment:
Let's have this be declared above getKeyCloakToken (alphabetical ordering)

Reply:
Fixed